target/linux/generic/config-filter

```diff
@@ -9,6 +9,7 @@ CONFIG_CLANG_VERSION=.*
 # CONFIG_INLINE_.* is not set
 # CONFIG_LD_.* is not set
 CONFIG_LLD_VERSION=.*
+CONFIG_PAHOLE_VERSION=*
```

Comment:
_looks_ like all the other lines in this file use `.*`. I have not checked what does the parsing to see if this matters.

Reply:
That had me interested as well, but having the dot or not it gets filtered out

Reply:
For the sake of consistency I would probably add the dot, in case this file ever gets script processed/parsed later on the expectation that the dot is there.

Reply:
Agree, updated to match others